addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -195,6 +195,10 @@ export class AddPageTemplatePreview extends Component {
             for (const imgEl of lazyLoadedImgEls) {
                 imgEl.setAttribute("loading", "lazy");
             }
+            if (!this.previewRef.el) {
+                // Stop the process when preview is removed
+                return;
+            }
             // Wait for fonts.
```

Comment:
@agau-odoo Is not this the same issue that @loco-odoo is fixing here ? (review I just added at https://github.com/odoo/odoo/pull/179865#pullrequestreview-2297611116)

I let you fight to know which PR to close, which version to target and which fix is right :crossed_swords: Louis, you probably win for the version, Alice probably wins a lot for the test... let's just see which code is best :open_mouth:

Reply:
Well to be fair, the reason I targeted master is because my test didn't fail in 17.4, but now I'm going to triple check that.

Reply:
@qsm-odo, @agau-odoo , if we want to add a test in this version, we would probably have to patch and add an artificial delay in the `onceAllImagesLoaded` utils in a test for it to fail without the fix. As this fix is needed for other Alice's PR and for the PR of url translation what do you think of merging this one and adding the test ASAP later? Would it be ok for you? :)

Reply:
Yes, given the rush, we can always add the test later. Just decide which fix is the best though ... :) One of you is stopping the function here, the other is making it ignore a delay :wink: Maybe @loco-odoo 's solution is the best here @agau-odoo ? :thinking:

Reply:
Yes I agree @loco-odoo's solution is better!

Reply:
Let's go with that for now then :)